### DISCUSSION TITLE
Detail video or document, How to play with this

### DISCUSSION THREAD

**Discussion by rainfall-kiran** [oobabooga/text-generation-webui#4276] [Q&A]
Any detail documentation or video , how to use it, because not getting the idea how play with it.

Like How to start, how to set model

I am new here so need it

**Comment by nbollman** [comment]
After following the detailed instructions for downloading and installing, launch the program for how you installed it...
Open the Text-Generation-WebUI with your web browser: [localhost:7860](url) (this should be the address)
Click on the "Model" tab at the top of the page.
Browse HuggingFace.co for various LLM models suitable for your hardware constraints (VRAM/RAM).
Choose a safe starting point like the 'Mistral-7b-Instruct model' from HuggingFace. [https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1)
Use the copy button to the right of the title 'mistralai/Mistral-7B-Instruct-v0.1' name, then switch back to your text-generation-webui paste it in the first line under 'Download model or LoRA'.
Download the model by clicking the download button after pasting the copied link.
Refresh the model list by clicking the blue arrow icon to the right of the model selector.
Select and load the downloaded model.
Start interacting with the LLM through the chat tab.
Please let me know if you need any further assistance or if you face any issues during these steps.

Let me know if this worked for you, if you have further questions try to be more specific, tell us about your hardware/software environment. Or where exactly you are having problems running the program.

**Reply by lombarthus** [reply]
Thank you, nbollman, for your detailed explanation on how to get things started with the UI! Much appreciated you put your time in explaining to us newbies.

I am using a GTX1070 with 8GB VRAM and the specified Mistral Model turns out to work really slow on my machine. Could you suggest any command line flags that would help speed things up a little?

Thank you!
Lombarthus

**Reply by missionfloyd** [reply]
The version linked above is way too big to fit in your VRAM. That's why it's so slow. Try this one instead. Under "Loader", select ExLlama.
https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GPTQ

In general, look for GPTQ models with 7B (parameters) or less (they'll be around 4GB), and load them with ExLlama or AutoGPTQ.
https://huggingface.co/TheBloke?search_models=gptq

**Reply by lombarthus** [reply]
Worked like a charm - thanks a bunch, @missionfloyd.

**Comment by rainfall-kiran** [comment]
Thank you so much for the response... I downloading this on Server, 
It was facing the issues.
1. Cloned the repository but in our server it was showing conda error..
So will look into it and let you know.
